Sources/NIO/EventLoopFuture.swift

```diff
@@ -1267,3 +1267,19 @@ func executeAndComplete<Value>(_ promise: EventLoopPromise<Value>?, _ body: () t
         promise?.fail(e)
     }
 }
+
+
+extension EventLoopFuture {
+    
```

Comment:
Minor nit: can we remove this whitespace?

Reply:
@Lukasa Done!